src/utils/api/handlerUtils.js

```diff
@@ -0,0 +1,13 @@
+const methodGET = R.compose(R.equals('GET'), R.propOr(null, "method"))
+const methodPOST = R.compose(R.equals('POST'), R.propOr(null, "method"))
+const methodPUT = R.compose(R.equals('PUT'), R.propOr(null, "method"))
+const methodDELETE = R.compose(R.equals('DELETE'), R.propOr(null, "method"))
+
+const resourceBase = R.compose(R.equals('/'), R.propOr(null, "resource"))
+export const createResourceTest = (resource) => R.compose(R.equals(resource), R.propOr(null, "resource"))
+
+// expect an object with resource and method keys e.g. { resource: '/', method: 'GET'g}
+export const getAPI = R.allPass([methodGET, resourceBase]);
```

Comment:
why does it needs to be base ('/') ? Also, you can define diff handlers for different verbs in lambda